### PR TITLE
chore: improve logging on slot transition and new block.

### PR DESCRIPTION
**Pull request by mpaulucci**

### CODE REVIEW

File: lib/lambda_ethereum_consensus/fork_choice/fork_choice.ex [hunk 1]
```diff
@@ -70,6 +70,11 @@ defmodule LambdaEthereumConsensus.ForkChoice do
   def handle_cast({:on_block, block_root, %SignedBeaconBlock{} = signed_block, from}, store) do
     slot = signed_block.message.slot
 
+    Logger.info("[Fork choice] Adding new block ",
+      root: block_root,
+      slot: slot
+    )
```
Comment (MegaRedHand) [inline lib/lambda_ethereum_consensus/fork_choice/fork_choice.ex:76]: Nit
```suggestion
    Logger.info("[Fork choice] Adding new block", root: block_root, slot: slot)
```

**Review by MegaRedHand — APPROVED** [review]
LGTM

File: lib/lambda_ethereum_consensus/fork_choice/fork_choice.ex (hunk 1 above)
Comment (mpaulucci) [reply lib/lambda_ethereum_consensus/fork_choice/fork_choice.ex:76]: what is the change? formatting?

Comment (MegaRedHand) [reply lib/lambda_ethereum_consensus/fork_choice/fork_choice.ex]: Yes. And a trailing space in the log message.

**Pull request merged**